MAUI/ProgressBar/Appearance.md

```diff
@@ -0,0 +1,392 @@
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar();
+circularProgressBar.Progress = 75;
+circularProgressBar.TrackRadiusFactor = 0.8;
+circularProgressBar.ProgressRadiusFactor = 0.75;
+circularProgressBar.TrackThickness = 0.05;
+circularProgressBar.ProgressThickness = 0.05;
+circularProgressBar.ThicknessUnit = SizeUnit.Factor;
+this.Content = circularProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %} 
+
+![appearance](images/appearance/circular-thickness.png)
+
+## Corner radius
+
+The `CornerRadius` property is used to customize the rounded edges in the linear progress bar as demonstrated in the following code sample.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<progressBar:SfLinearProgressBar Progress="50"
+                                 TrackHeight="10" 
+                                 TrackCornerRadius="5"
+                                 ProgressHeight="10"
+                                 ProgressCornerRadius="5">
+    
+</progressBar:SfLinearProgressBar>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearProgressBar linearProgressBar = new SfLinearProgressBar();
+linearProgressBar.Progress = 50;
+linearProgressBar.TrackHeight = 10;
+linearProgressBar.ProgressHeight = 10;
+linearProgressBar.ProgressCornerRadius = 5;
+linearProgressBar.TrackCornerRadius = 5;
+this.Content = linearProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %} 
+
+![cornerradius](images/appearance/corner-radius.png)
+
+## Corner style customization
+
+The `ProgressCornerStyle` and `TrackCornerStyle` property of circular progress bar specifies the corner type for the progress and track. The corners can be customized using the `BothFlat`, `BothCurve`, `StartCurve`, and `EndCurve` options. The default value of this property is `BothFlat`.
+
+The following code sample demonstrates the corner style customization in progress and track indicator.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<progressBar:SfCircularProgressBar Progress="50"
+                                   TrackCornerStyle="BothCurve"
+                                   ProgressCornerStyle="BothCurve"
+                                   StartAngle="180"
+                                   EndAngle="360">
+    
+</progressBar:SfCircularProgressBar>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar();
+circularProgressBar.Progress = 50;
+circularProgressBar.TrackCornerStyle = CornerStyle.BothCurve;
+circularProgressBar.ProgressCornerStyle = CornerStyle.BothCurve;
+circularProgressBar.StartAngle = 180;
+circularProgressBar.EndAngle = 360;
+this.Content = circularProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %} 
+
+![cornerstyle](images/appearance/corner-style.png)
+
+## Color customization
+
+The following properties are used to customize the color in the progress bar:
+
+* `ProgressFill`: Represents the color of the progress indicator.
```

Comment:
* `ProgressFill:` Represents the color of the progress indicator.

Reply:
done